Example/android/app/build.gradle

```diff
@@ -80,38 +69,19 @@ def enableProguardInReleaseBuilds = false
  */
 def jscFlavor = 'org.webkit:android-jsc:+'
 
-/**
- * Private function to get the list of Native Architectures you want to build.
- * This reads the value from reactNativeArchitectures in your gradle.properties
- * file and works together with the --active-arch-only flag of react-native run-android.
- */
-def reactNativeArchitectures() {
-    def value = project.getProperties().get("reactNativeArchitectures")
-    return value ? value.split(",") : ["armeabi-v7a", "x86", "x86_64", "arm64-v8a"]
-}
-
 android {
     ndkVersion rootProject.ext.ndkVersion
+    buildToolsVersion rootProject.ext.buildToolsVersion
+    compileSdk rootProject.ext.compileSdkVersion
 
-    compileSdkVersion rootProject.ext.compileSdkVersion
-
-    namespace "com.swmansion.reanimated.example"
+    namespace "com.example"
```

Comment:
Do we really want to change the namespace here? I'm okay with that change (it's better to have our app aligned our app with the official template).

Reply:
Yeah, I didn't like all these shenanigans needed to change `Example` to `ReanimatedExample` on both `iOS` and `Android` so I'd remove it as a whole.

Reply:
Okay, this makes perfect sense. Thanks